_posts/2019-04-01-multi-instance-aspnetcore-2.2-data-protection-using-redis-and-an-azure-key-vault-certificate.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: Multi Instance ASP.NET Core 2.2 Data Protection Using Redis and an Azure Key Vault Certificate
+date: 2019-04-01 10:00:00
+tags:
+- .NET
+- asp.net
+- asp.net core
+categories:
+- asp.net
+- asp.net core
+- development
+twitter_text: "Multi Instance #aspnetcore 2.2 Data Protection Using #redis and an @Azure Key Vault Certificate"
+authors: Ken Dale
+image: https://images.unsplash.com/photo-1504253163759-c23fccaebb55?ixlib=rb-1.2.1&auto=format&fit=crop&w=1950&q=80
+image_url: https://unsplash.com/photos/IM8ZyYaSW6g
+image_credit: Vladimir Anikeev
+---
+
+We're in the process of upgrading our infrastructure to run our Azure Web Apps in multiple datacenter locations behind Azure Traffic Manager instances. Running multiple instances without using the scale up slider means you're responsible for any server side state that needs to be shared between instances *(or, ensure that traffic always returns to the same instance repeatedly, but we don't want to do that...)*.
```

Comment:
You may want to mention why we are doing this. You mention it at the end of the article but not here. It might be good to have the two bits tie back to each other.